api/src/main/resources/org/openmrs/api/db/hibernate/Encounter.hbm.xml

```diff
@@ -59,7 +59,7 @@
 			<column name="form_id" />
 		</many-to-one>
 		<!-- bi-directional one-to-many association to Order -->
-		<set name="orders" inverse="true">
+		<set name="orders" inverse="true" where="order_group_id is null">
```

Comment:
You are correct it should return all the orders irrespective of order_groups.
 That condition for order_group_id is null, in encounter mapping file is now removed.
 I missed to remove it from the mapping file. Modified and pushed it now.